src/notebooks/controllers/notebookIPyWidgetCoordinator.ts

```diff
@@ -159,30 +149,19 @@ export class NotebookIPyWidgetCoordinator implements IExtensionSyncActivationSer
             );
             return;
         }
-        traceVerbose(`Intiailize notebook communications for editor ${getDisplayPath(editor.notebook.uri)}`);
+        traceVerbose(`Initialize notebook communications for editor ${getDisplayPath(editor.notebook.uri)}`);
         const comms = new NotebookCommunication(editor, controller);
         this.addNotebookDisposables(notebook, [comms]);
         this.notebookCommunications.set(editor, comms);
-        const { token } = new CancellationTokenSource();
-        this.resolveKernel(notebook, comms, token).catch(noop);
-    }
-    private resolveKernel(
```

Comment:
This was unnecessarily async, and an unnecessary method as well. Just made all of this inline to make it easier to follow the code.